docs/source/install.rst

```diff
@@ -51,19 +51,13 @@ The instructions above install a CPU-only version of SCICO. To install a version
 
 1. Follow the CPU only instructions, above
 
-2. Identify which version of jaxlib was installed
-
-   ::
-
-      pip list | grep jaxlib
-
-3. Install the same version of jaxlib, but with GPU support.
-   For help with this, see `JAX with GPU support <https://github.com/google/jax#installation>`_.
+2. Install the version of jaxlib with GPU support, as described in the `JAX installation
+   instructions  <https://github.com/google/jax#installation>`_.
    The command will be something like
 
    ::
 
-      pip install --upgrade "jaxlib==0.3.0+cuda11.cudnn805" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
+      pip install -upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```

Comment:
Careful. I believe that the new command upgrades both jax and jaxlib, possible to outside the version range supported by SCICO. The intent of the old command was to get the GPU version of jaxlib at the same version number that was already installed, ensuring compatibility with SCICO. But I can't say for sure that it still works.

Reply:
Good point. I will add a warning on this.